src/main/java/dk/sdu/se_f22/sortingmodule/range/rangepublic/RangeFilterFilterResults.java

```diff
@@ -20,11 +20,11 @@ public class RangeFilterFilterResults {
      * using the filters specified in rangeFilters param.
      */
     public static SearchHits filterResults(SearchHits searchHits, List<RangeFilter> rangeFilters) throws IllegalImplementationException {
-        Collection productHits = searchHits.getProducts();
+        Collection Products = searchHits.getProducts();
```

Comment:
variable name change? Also does not follow common naming scheme.

Reply:
We (group 4.4) would appreciate that the variable is not changed. 
It is not a ClassReference and thus does not have an impact on your module.

Reply:
think they ran some project wide search and replace without being smart about it